src/app/core/services/data.service.ts

```diff
@@ -57,12 +65,12 @@ export class DataService {
    * @param csvInput - string containing csv data.
    * @return admins - object that represents parsed csv data.
    */
-  private parseAdminAllocation(csvInput: string): {} {
+  private parseAdminAllocation(csvInput: string): Admins {
     // CSV Headers
     const NAME = 'name';
     const ROLE = 'role';
 
-    const admins = {};
+    const admins: Admins = {};
     let parsedCSV: {}[];
     parsedCSV = this.csvParser(csvInput);
```

Comment:
```suggestion
    const parsedCSV: ParsedUserData[] = this.csvParser(csvInput);
```

Given we have already defined the return type of `csvParser`, we can make use of this in future declarations of 
`parsedCSV`. What are your thoughts? @zatkiller

Reply:
I agree, that would make variables clearer to other readers too